Library/Homebrew/cmd/deps.rb

```diff
@@ -140,12 +140,12 @@ def puts_deps_tree(formulae)
 
   def recursive_deps_tree(f, prefix)
     reqs = f.requirements.select(&:default_formula?)
+    deps = f.deps.default
     max = reqs.length - 1
     reqs.each_with_index do |req, i|
-      chr = i == max ? "└──" : "├──"
+      chr = i == max && deps.empty? ? "└──" : "├──"
```

Comment:
I'd make this an `if`/`else` as there's a bit too much logic going on here (even before, arguably) for a ternary operator.